### PR TITLE
Update SE clocks example for Reference-FMUs

### PR DESCRIPTION
**Pull request by MBlesken**
Update SE clocks example for Reference-FMUs for FMI 3.0 alpha 6

### CODE REVIEW

File: Clocks/FMI3.xml [hunk 1]
```diff
@@ -1,42 +1,39 @@
...
+  <Int32 name="inClock2Ticks" valueReference="2002" causality="output"/>
+  <Int32 name="inClock3Ticks" valueReference="2003" causality="output"/>
+  <Int32 name="totalInClockTicks" valueReference="2004" causality="output"/>
+  <Int32 name="result2" valueReference="2005" causality="output"/>
+  <Int32 name="input2" valueReference="2006" causality="input" start="0"/>
+  <Int32 name="output3" valueReference="2007" causality="output"/>
+ </ModelVariables>
+ <ModelStructure>
+  <Output valueReference="1005"/>
+  <Output valueReference="2001" dependencies="1001"/>
```
Comment (chrbertsch) [inline Clocks/FMI3.xml:24]: Do we really list clocks in the list of dependencies of outputs?
We have to specify these dependenencies in the "clock" element in the model struture, which is missing currently

Comment (MBlesken) [reply Clocks/FMI3.xml:24]: Thanks for checking so carefully. I will adjust this ...

Comment (chrbertsch) [reply Clocks/FMI3.xml:24]: Thanks for fixing. 
Just saw, that in https://fmi-standard.org/docs/3.0-dev/#ModelStructure it is clearly defined that output clocks should be listed as outtputs as you did

**Review by chrbertsch — CHANGES_REQUESTED** [review]
The example runners are still called scs_synchronous.c  and examples/scs_threaded.c, should be change to se_...

**Comment by MBlesken** [comment]
> The example runners are still called scs_synchronous.c and examples/scs_threaded.c, should be change to se_...
@chrbertsch I think that @t-sommer wants to adjust this and similar adjustments all at one by himself.

**Comment by MBlesken** [comment]
@t-sommer I would like to invite you to the review as well. I admit that my last programming experiences have been a while ago so a appreciate any comments that help me improve. It would also be great if you could take a look at the two failed test. I'm not sure if I can reproduce these with the tools I have available.

**Comment by t-sommer** [comment]
Can change this PR to „Ready for Review“?

File: Clocks/FMI3.xml [hunk 2]
```diff
@@ -1,51 +1,43 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <fmiModelDescription fmiVersion="3.0-alpha.6" modelName="Clocks" instantiationToken="{8c4e810f-3df3-4a00-8276-176fa3c9f000}">
```
Comment (chrbertsch) [inline Clocks/FMI3.xml:2]: We should refer to alpha.7

File: Clocks/FMI3.xml [hunk 3]
```diff
@@ -1,51 +1,43 @@
...
+ </ModelVariables>
+ <ModelStructure>
+  <Output valueReference="1005" dependencies="1001 1002 1003"/>
+  <Output valueReference="2001" dependencies="1001"/>
+  <Output valueReference="2002" dependencies="1002"/>
+  <Output valueReference="2003" dependencies="1003"/>
+  <Output valueReference="2004" dependencies="1001 1002 1003"/>
+  <Output valueReference="2005" dependencies="1002 2006"/>
+  <Output valueReference="2007" dependencies="1003"/>
+  <Clock valueReference="1001" clockedVariables="1005 2001 2004"/>
```
Comment (chrbertsch) [inline Clocks/FMI3.xml:30]: Fine for Alpha.7, but should be changed for alpha.8 or beta.1 due to https://github.com/modelica/fmi-standard/pull/1375

File: src/fmi3Functions.c [hunk 4]
```diff
@@ -865,7 +865,17 @@ fmi3Status fmi3GetIntervalDecimal(fmi3Instance instance,
...
+    for (size_t i = 0; i < nValueReferences; i++) {
+        Status s = getInterval(instance, valueReferences[i], &interval[i], (int*)&qualifier[i]);
+        status = max(status, s);
+        if (status > Warning) return (fmi3Status)status;
+    }
+
+    return (fmi3Status)status;
 }
 
 fmi3Status fmi3SetIntervalDecimal(fmi3Instance instance,
```
Comment (chrbertsch) [inline src/fmi3Functions.c:881]: The argument names  of thes function changed in https://github.com/modelica/fmi-standard/pull/1361/files

**Review by chrbertsch — CHANGES_REQUESTED** [review]
Looks good
Minor changes should be made to fit to alpha.7
Would it be possible to create a short description in a README.md?
@t-sommer : can the readme.html be automaticall generated from README.md? Can this be then used as the index.html in the /documentation folder of the FMU?

**Comment by t-sommer** [comment]
> Would it be possible to create a short description in a README.md?
> @t-sommer : can the readme.html be automaticall generated from README.md? Can this be then used as the index.html in the /documentation folder of the FMU?

This should already be the case.

**Comment by MBlesken** [comment]
@chrbertsch @t-sommer thank you for your comments and help.

**Pull request merged**